### PR TITLE
#5435 add rmrk label

### LINKED ISSUE #5435: Show on landing in latest sales & listing RMRK2 in card
To indicate type of network, RMRK2 items missing rmrk2 in corner as we have Kusama or Basilisk

image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5435
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-03-30 at 8 23 16 PM

### CODE REVIEW

File: components/carousel/module/CarouselInfo.vue [hunk 1]
```diff
@@ -68,6 +68,7 @@ const chainName = computed(() => {
     rmrk: 'Kusama',
     snek: 'Snek (Rococo)',
     bsx: 'Basilisk',
+    rmrk2: 'Rmrk2',
```
[the remark this answers is not in the document]
Comment (prachi00) [reply components/carousel/module/CarouselInfo.vue]: done

**Review by roiLeo — COMMENTED** [review]
I thought we had replaced `rmrk` by `ksm`?

**Comment by prachi00** [comment]
> 

I checked value for props.item.chain , it was rmrk

**Comment by prachi00** [comment]
> please fix ts error, you can check or reproduce it by running `pnpm prepare`

done

File: libs/static/src/types.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ export type Prefix = 'bsx' | 'glmr' | 'ksm' | 'movr' | 'rmrk2' | 'snek'
 // | 'statemine'
 // | 'westmint'
 
-export type BackwardPrefix = 'rmrk' & Prefix
+export type BackwardPrefix = 'rmrk' | Prefix
```
[the remark this answers is not in the document]
Comment (prachi00) [reply libs/static/src/types.ts:5]: this would break types though

**Comment by yangwao** [comment]
@prachi00 let's check this one 💪

**Comment by yangwao** [comment]
let's see if it builds properly 👀

**Comment by yangwao** [comment]
@vikiival 👀

**Comment by vikiival** [comment]
> @vikiival 👀

[4da142b](https://github.com/kodadot/nft-gallery/pull/5438/commits/4da142b8d4015e43fbdd51f59f755432e27b6821)

**Comment by yangwao** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 33.1 USD/KSM ~ 0.302 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xb63002e270bb74fdd547cce38d11020e742896b5407afe4edd232bbf597f750c](https://kusama.subscan.io/extrinsic/0xb63002e270bb74fdd547cce38d11020e742896b5407afe4edd232bbf597f750c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
oh @prachi00 seems doesn't work 
image

**Pull request merged**